app/models/journeys/teacher_student_loan_reimbursement/session_answers_type.rb

```diff
@@ -0,0 +1,38 @@
+module Journeys
+  module TeacherStudentLoanReimbursement
+    class SessionAnswersType < ActiveModel::Type::Value
```

Comment:
I left them separate to avoid any meta programmy type stuff as they each need to access the `SessionAnswers` class in their respective modules. I'm not opposed to moving to a single SessionAnswersType and doing something with `module_parent` to get at the correct `SessionAnswers` class if you think that's better. Thought I think these SessionAnswersType classes are unlikely to be changed. What do you think?